src/proto/api_messages_test.js

```diff
@@ -56,6 +60,20 @@ function isEqual(thisArray, otherArray) {
 
 describe('api_messages', () => {
 
+  describe('test_AlreadySubscribedResponse', () => {
+    it('should deserialize correctly', () => {
+      const /** !AlreadySubscribedResponse  */ alreadysubscribedresponse = new AlreadySubscribedResponse();
```

Comment:
Would it be easy to make these local test variables camelCase?

Reply:
This comment is more of a nit, for readability

Reply:
Sure please feel free to bug on Protobuf Library generator code and assign to me. Would be happy to address it during fix it.

Reply:
Sounds good